engine/src/flutter/impeller/entity/shaders/glyph_atlas.vert

```diff
@@ -14,7 +16,15 @@ in vec2 position;
 
 out vec2 v_uv;
 
+vec4 impRound(vec4 value) {
+  return floor(value + vec4(0.5));
+}
+
 void main() {
   gl_Position = frame_info.mvp * vec4(position, 0, 1);
+  if (frame_info.should_round > 0) {
```

Comment:
Won't this also round the depth values?

Reply:
You can change this to:

```
vec4 impRound(vec4 value) {
  return vec4(floor(value.xy + vec2(0.5)), value.zw);
}
```

To fix it

Reply:
yep, thanks